scripts/build-appbuilder.sh

```diff
@@ -3,6 +3,81 @@ SENTRY_CLI="node_modules/.bin/sentry-cli"
 SENTRY_ORG="shapediver"
 SENTRY_PROJECT="app-builder"
 
+# Function to build and deploy the app
+# $1: deploy - should we deploy the build?
+# $2: prefix - the prefix for the deployment, either "v1/main" or "app/builder/v1/main"
+# $3: version - the version of the deployment (branch name or npm version)
+# $4: deploying_branch - 1 if we are deploying a branch, 0 if we are deploying a version
+build_and_deploy() {
+    deploy=$1
+    prefix=$2
+    version=$3
+    deploying_branch=$4
+
+    # Check if sentryconfig.local.ts exists, copy it to sentryconfig.ts if it does
+    build_timestamp=$(date +'%Y-%m-%d_%H:%M')
+    if [ -f "sentryconfig.local.ts" -a ! -z "${deploy}" ]; then
+        echo "Configuring sentry for build timestamp: ${build_timestamp}"
+        mv sentryconfig.ts sentryconfig.ts.bak
+        sed -e "s/BUILD_TIMESTAMP/${build_timestamp}/" sentryconfig.local.ts > sentryconfig.ts
+    fi
+
+    echo "Building AppBuilder version $version with prefix $prefix"
+    vite build --base=$prefix/$version/
+    if [ $? -ne 0 ]; then
+        echo "Build failed."
+        exit 1
+    fi
+
+    if [ -z "${deploy}" ]; then
+        echo "Skipping deployment."
+        exit 0
+    fi
+
+    if [ -z "$APPBUILDER_BUCKET" ]; then
+        echo "APPBUILDER_BUCKET environment variable is not set."
+        exit 1
+    fi
+
+    cachecontrol="public, max-age=31536000, immutable"
+    # if we are deploying a branch, we need to change the cache control, compared to a version
+    if [ $deploying_branch -eq 1 ]; then
+        cachecontrol="public, max-age=0, s-maxage=86400"
+    fi
+
+    echo "Deploying to version $version with prefix $prefix to bucket $APPBUILDER_BUCKET"
+
+    # depending on the prefix, we need to deploy to different locations
+    if [ $prefix == "v1/main" ]; then
```

Comment:
@MajorMeerkatThe3rd I suggest to introduce a single variable for `main`, defined on top of the script. This will allow us to copy this script to the other repos (for ijewel3d and peri) and reuse it with minimal changes.